cuebot/src/main/java/com/imageworks/spcue/servant/ManageJob.java

```diff
@@ -357,9 +363,15 @@ public void resume(JobResumeRequest request, StreamObserver<JobResumeResponse> r
     public void setMaxCores(JobSetMaxCoresRequest request, StreamObserver<JobSetMaxCoresResponse> responseObserver) {
         try{
             setupJobData(request.getJob());
-            jobDao.updateMaxCores(job, Convert.coresToWholeCoreUnits(request.getVal()));
-            responseObserver.onNext(JobSetMaxCoresResponse.newBuilder().build());
-            responseObserver.onCompleted();
+            if (isJobFinished()) {
```

Comment:
If it's not too much trouble, it would be great to add some unit tests that confirm the new config settings is applied correctly.